src/profile/ResetPassword.js

```diff
@@ -1,148 +1,251 @@
-  const [state, setState] = useState({
-    code: "",
-    email: "",
-    password: "",
-    codeSent: false,
-    confirmed: false,
-    confirmPassword: "",
-    isConfirming: false,
-    isSendingCode: false,
-  });
-
-  const validateEmailForm = () => state.email.length > 0;
-
-  const validateResetForm = () =>
-    state.code.length > 0 &&
-    state.password.length > 0 &&
-    state.password === state.confirmPassword;
-
-  const handleChange = (event) => {
-    setState({
-      ...state,
-      [event.target.id]: event.target.value,
-    });
-  };
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingTop: theme.spacing(5),
+    width: 300,
+
+    "& .css-36njyd-MuiInputBase-root-MuiFilledInput-root": {
+      backgroundColor: theme.palette.background.paper,
+    },
+    "& .MuiTextField-root": {
+      width: 300,
+    },
+    "& .MuiFormLabel-root": {
+      fontSize: 16,
+      color: theme.palette.primary.main,
+    },
+    "& .MuiInputBase-input": {
+      fontSize: 16,
+    },
+    "& .MuiButtonBase-root": {
+      marginTop: theme.spacing(1),
+      fontSize: 16,
+    },
+    "& .error": {
+      fontSize: 14,
+      color: "rgb(220, 66, 45)",
+    },
+    "& .MuiFormHelperText-root": {
+      fontSize: 16,
+      color: "#808080",
+    },
+  },
+}));
+
+const ResetPassword = ({ initialFetch, setCloseLoading, setLoading }) => {
+  const classes = useStyles();
+
+  const [isConfirming, setIsConfirming] = useState(false);
+  const [isSendingCode, setIsSendingCode] = useState(false);
+  const [codeSent, setCodeSent] = useState(false);
+  const [confirmed, setConfirmed] = useState(false);
+  const [disabled] = useState(false);
 
-  const handleSendCodeClick = async (event) => {
-    event.preventDefault();
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    getValues,
+  } = useForm();
 
-    setState({ ...state, isSendingCode: true });
+  const {
+    register: register2,
+    formState: { errors: errors2 },
+    handleSubmit: handleSubmit2,
+    watch,
+  } = useForm();
+
+  let password1 = watch("password");
+  let email1 = getValues("email");
+
+  const handleSendCodeClick = async (data) => {
+    setLoading();
+
+    setIsSendingCode(true);
 
     try {
-      await Auth.forgotPassword(state.email);
-      setState({ ...state, codeSent: true });
+      await Auth.forgotPassword(data.email);
+
+      setCodeSent(true);
+      setCloseLoading();
     } catch (e) {
       alert(e.message);
-      setState({ ...state, isSendingCode: false });
+      setIsSendingCode(false);
+      setCloseLoading();
     }
   };
 
-  const handleConfirmClick = async (event) => {
-    event.preventDefault();
+  const handleConfirmClick = async (data) => {
+    setLoading();
 
-    setState({ ...state, isConfirming: true });
+    setIsConfirming(true);
 
     try {
-      await Auth.forgotPasswordSubmit(state.email, state.code, state.password);
-      setState({ ...state, confirmed: true });
+
```

Comment:
Can you please remove this unnecessary line space? In the next line after the `try` block, there should be some code - shouldn't be any space. My `prettier` extension is complaining.